From: paul cannon <paul@spacemonkey.com>
Date: Thu, 3 Apr 2014 11:27:09 -0600
Subject: don't use ez_setup

Debian packages aren't supposed to download stuff while building, and
since the version of setuptools in stable is less than the one ez_setup
wants, and since some system python packages don't ship their .egg-info
directories, it might try.

It's ok though, we can rely on the Depends and Build-Depends for making
sure python-setuptools and the various other deps are around at the right
times.
---
 setup.py | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/setup.py b/setup.py
index 0c28d3d..c0fd6c1 100644
--- a/setup.py
+++ b/setup.py
@@ -1,8 +1,5 @@
 import sys
 
-import ez_setup
-ez_setup.use_setuptools()
-
 if __name__ == '__main__' and sys.argv[1] == "gevent_nosetests":
     from gevent.monkey import patch_all
     patch_all()
@@ -174,8 +171,8 @@ def run_setup(extensions):
         author_email='tyler@datastax.com',
         packages=['cassandra', 'cassandra.io'],
         include_package_data=True,
-        install_requires=dependencies,
-        tests_require=['nose', 'mock', 'ccm', 'unittest2', 'PyYAML', 'pytz'],
+        install_requires=(),
+        tests_require=(),
         classifiers=[
             'Development Status :: 5 - Production/Stable',
             'Intended Audience :: Developers',
